fixup-mountpoints

```diff
@@ -1775,6 +1775,17 @@ case "$DEVICE" in
             -e 's block/bootdevice/by-name/modem sde7 ' \
             -e 's block/bootdevice/by-name/bluetooth sde16 ' \
             "$@"
+        ;;    
+        
+    "mata")
+        sed -i \
+            -e 's block/bootdevice/by-name/system /dev/block/platform/soc/1da4000.ufshc/by-name/system ' \
+	    -e 's block/bootdevice/by-name/userdata /dev/block/sda16 ' \
+	    -e 's block/bootdevice/by-name/modem /dev/block/platform/soc/1da4000.ufshc/by-name/modem ' \
+	    -e 's block/bootdevice/by-name/bluetooth /dev/block/platform/soc/1da4000.ufshc/by-name/bluetooth ' \
+	    -e 's block/bootdevice/by-name/dsp /dev/block/platform/soc/1da4000.ufshc/by-name/dsp ' \
+	    -e 's block/bootdevice/by-name/persist /dev/block/sda4 ' \
```

Comment:
This won't work because you replace the relative part `block/bootdevice/...` with a absolute path `/dev/block/...`.
This means that `/dev/block/bootdevice/by-name/persist` will be replaced with `/dev/dev/block/sda4`.

Reply:
I see what you're saying. I took the /dev/ out of the returns as well and made them relative and switched userdata and persist to be inline with the others in the file.

I'm not sure if there's a way to address the other returns as mata is an A/B device (and seems to be the first in here). If you can point me in the direction of some reading materials, I'm happy to learn and fix the rest of them as needed!